array_interpolate_linear.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "array_interpolate_linear",
     "summary": "One-dimensional linear interpolation for arrays",
-    "description": "Performs a linear interpolation for each of the no-data values (`null`) in the array given, except for leading and trailing no-data values.\n\nThe linear interpolants are defined by the array indices or labels (x) and the values in the array (y).",
+    "description": "Performs a linear interpolation for each of the NaN and no-data values in the array given, except for leading and trailing NaN and no-data values.\n\nThe linear interpolants are defined by the array indices or labels (x axis) and the values in the array (y axis).",
```

Comment:
> NaN and no-data

Isn't it easier to just speak about no-data values and consider NaN to be a case of no-data (e.g. ina IEEE float context). This note on NaN might be left implicit for reader, or mentioned explicitly.

Reply:
no-data and NaN may be different values, so I think it's better to make it explicit.